### ISSUE TITLE
to become responsive

### ISSUE THREAD

**Issue by stefano-buscaglia** [jberger/Galileo#48]
hi,
i suggest you to switch to bootstrap >= 3 that is responsive. 

I know Galileo is small CMS for small project, but today even a small project is better if it's mobile ready :)

If for some reason you prefer to stay on B 2.1 i can pull a small css that overwrite few css rules and make the actual template responsive. 

have a nice day

Stefano

**Comment by jberger** [comment]
Yes I should get around to that. Galileo doesn't get enough of my time lately :s.
